ts/packages/ts/src/commands/EvalCommand.ts

```diff
@@ -291,7 +291,10 @@ export class EvalCommand extends Command<EvalCommandReturn> {
       ...(await this.evalFromStoreSet(
         _.pickBy(
           result,
-          (current) => current.result.origin === EvaluatedConfigOrigin.EmptyValue && !current.context.cfgu.template,
+          (current) =>
+            current.result.origin === EvaluatedConfigOrigin.EmptyValue &&
+            !current.context.cfgu.template &&
+            !current.context.cfgu.lazy,
```

Comment:
this should already been done on the `evalFromConfigsOverride` no?
you should validate somewhere that all lazy configs are set via the config override.

Reply:
![Uploading image.png…]()

Reply:
@RonConfigu image is broken here